src/test/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperationsTest.scala

```diff
@@ -668,6 +668,45 @@ class TezosDatabaseOperationsTest
       BlocksRow(1,1,"R0NpYZuUeF",new Timestamp(1),0,"fitness",Some("context1"),Some("sigTZ2IB879wD"),"protocol",Some("YLBMy"),"aQeGrbXCmG",None)
     )
 
+    "get all values from the table with nulls as nones" in {
+
+      val columns = List()
+
+      val populateAndTest = for {
+        _ <- Tables.Blocks ++= blocksTmp
+        found <- sut.selectWithPredicates(Tables.Blocks.baseTableRow.tableName, columns, List.empty, List.empty, 3)
+      } yield found
+
+      val result = dbHandler.run(populateAndTest.transactionally).futureValue
+      result shouldBe List(
+        Map("operations_hash" -> None, "timestamp" -> Some(new Timestamp(0)), "context" -> Some("context0"), "proto" -> Some(1),
```

Comment:
cosmetic: would you kindly indent these results, to help any further change here with enhanced readability?

Reply:
done